src/PowerShellEditorServices/Services/DebugAdapter/Debugging/VariableDetails.cs

```diff
@@ -414,4 +428,19 @@ public override string ToString()
             }
         }
     }
+
+    /// <summary>
+    /// A VariableDetails that only returns the raw view properties of the object, rather than its values.
+    /// </summary>
+    internal class VariableDetailsRawView : VariableDetails
+    {
+        private const string RawViewName = "Raw View";
+        public VariableDetailsRawView(object value) : base(RawViewName, value) { }
```

Comment:
```suggestion
        public VariableDetailsRawView(object value) : base(RawViewName, value)
        {
        }

```

I personally don't mind how you have it, but once we set up a formatter they tend to complain. Also new line to separate